modules/oc-mirror-signature-mirroring-procedure.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-signature-mirroring-procedure_{context}"]
+= Enabling signature mirroring for oc-mirror plugin v2
+
+By default, signature mirroring is disabled. You can enable signature mirroring for all images by setting the `--remove-signatures=false` flag.
+
+When enabled, `oc-mirror` mirrors `Sigstore` tag-based signatures for the following images:
+
+* {product-title} release images
+* Operator images
+* Additional images
+* Helm charts
+
+[NOTE]
+====
+If you do not provide a configuration file, the `oc-mirror` plugin v2 enables signature mirroring for all images by default when the `--remove-signatures=false` flag is used.
```

Comment:
Same thing here, I don't think the full tool name should use backticks, since that formatting is reserved for elements like actual command or file syntax:
```suggestion
If you do not provide a configuration file, the oc-mirror plugin v2 enables signature mirroring for all images by default when the `--remove-signatures=false` flag is used.
```